src/Components/WebView/Platforms/WindowsForms/src/BlazorWebView.cs

```diff
@@ -36,9 +36,20 @@ public BlazorWebView()
             {
                 Dock = DockStyle.Fill,
             };
-            Controls.Add(_webview);
+            ((BlazorWebViewControlCollection)Controls).AddInternal(_webview);
         }
 
+        /// <summary>
+        /// Returns the inner <see cref="WebView2Control"/> used by this control.
+        /// </summary>
+        /// <remarks>
+        /// Directly using some functionality of the inner web view can cause unexpected results because its behavior
+        /// is controlled by the <see cref="BlazorWebView"/> that is hosting it.
+        /// </remarks>
+        [Browsable(false)]
```

Comment:
Is this equivalent to `[EditorBrowsable(EditorBrowsable.Never)]`? If so that seems quite strict. I'd be open to letting people discover this more easily.

Reply:
Oh I guess it's something to do with the WinForms editor. If so please ignore this!

Reply:
Right, it just hides it from the property grid (and the DesignerSerialization thing prevents code-gen from trying to put something in `Form1.designer.cs`).